src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -528,8 +528,7 @@ public OpenIdConnectAuthenticationHandler(HttpClient backchannel)
             var subjectClaimType = identity.FindFirst(ClaimTypes.NameIdentifier);
```

Comment:
this may cause issues if the 'ClaimTypes.NameIdentifier' does not map to sub. It would be less fragile to get the 'sub' claim from the original 'id_token'.